plugins/Referrers/Controller.php

```diff
@@ -87,19 +88,58 @@ public function getEvolutionGraph($typeReferrer = false, array $columns = array(
             if ($typeReferrer === false) {
                 $typeReferrer = Common::getRequestVar('typeReferrer', Common::REFERRER_TYPE_DIRECT_ENTRY);
             }
-            $label = self::getTranslatedReferrerTypeLabel($typeReferrer);
-            $total = $this->translator->translate('General_Total');
 
             if (!empty($view->config->rows_to_display)) {
                 $visibleRows = $view->config->rows_to_display;
             } else {
-                $visibleRows = array($label, $total);
+                $visibleRows = array($typeReferrer, 'total');
             }
 
-            $view->requestConfig->request_parameters_to_modify['rows'] = $label . ',' . $total;
+            $view->requestConfig->request_parameters_to_modify['rows'] = $typeReferrer . ',total';
         }
-        $view->config->row_picker_match_rows_by = 'label';
-        $view->config->rows_to_display = $visibleRows;
+
+        // We need to configure a custom set of selectable rows
```

Comment:
Would this comment still be needed if the filter was extracted and easier to understand from reading the code?

Reply:
I could try to adjust how evolution charts work in general. So it would be possible to handle them with another column as identifier. I'll give that a quick try, but might stop it again if it would end up in a too big refactoring...